src/components/WebChat/WebChat.css

```diff
@@ -0,0 +1,130 @@
+.webchat-container{
+  display: flex; 
+  flex-direction: column;
+  position: fixed;
+  right: 0;
+  bottom: 0;
+  bottom: 20px;
+  box-shadow: 5px 5px 20px #fff;
+}
+
+.webchat-panel-inner{
+  flex-direction: column;
```

Comment:
not sure what this line does. without applying "flex" the "flex-direction" will not take effect anyway.

Reply:
The element has another class which has display flex